UI/Web/src/app/nav/_components/grouped-typeahead/grouped-typeahead.component.scss

```diff
@@ -1,5 +1,5 @@
 form {
-    max-height: 38px;
+    /*max-height: 38px;*/
```

Comment:
Unless unsure if needed elsewhere, do not comment out code just go ahead and remove it.

Reply:
Understood, removing it